xero/auth.py

```diff
@@ -336,7 +336,7 @@ class PartnerCredentials(PublicCredentials):
         oauth_authorization_expires_at tells when the overall access
         permissions expire (~10 year window)
     """
-    def __init__(self, consumer_key, consumer_secret, rsa_key, client_cert,
+    def __init__(self, consumer_key, consumer_secret, rsa_key, client_cert=None,
```

Comment:
The docstring of this class should be updated regarding `client_cert`. It should say that this argument is no more needed regarding new Xero security policy, but stays for now to be backward-compatible.